kse/src/main/java/org/kse/gui/dialogs/DViewSignedJar.java

```diff
@@ -0,0 +1,360 @@
+            TableColumn column = jtJarEntries.getColumnModel().getColumn(i);
+
+            column.setPreferredWidth(tableModel.getColumnSize(i));
+            if (i == SignedJarTableModel.COL_DATE) {
+                column.setCellRenderer(new DateRenderer());
+            }
+        }
+
+        ListSelectionModel listSelectionModel = jtJarEntries.getSelectionModel();
+        listSelectionModel.addListSelectionListener(evt -> {
+            if (!evt.getValueIsAdjusting())  { // Ignore spurious events
+                try {
+                    CursorUtil.setCursorBusy(DViewSignedJar.this);
+                    entrySelection();
+                } finally {
+                    CursorUtil.setCursorFree(DViewSignedJar.this);
+                }
+            }
+        });
+
+        jtJarEntries.addMouseListener(new MouseAdapter() {
+            @Override
+            public void mouseClicked(MouseEvent evt) {
+                maybeDisplayJarEntryCertificates(evt);
+            }
+        });
+
+        jspJarEntryTable = PlatformUtil.createScrollPane(jtJarEntries,
+                                                             ScrollPaneConstants.VERTICAL_SCROLLBAR_AS_NEEDED,
+                                                             ScrollPaneConstants.HORIZONTAL_SCROLLBAR_AS_NEEDED);
+        jspJarEntryTable.getViewport().setBackground(jtJarEntries.getBackground());
+        jspJarEntryTable.setPreferredSize(new Dimension(600, 200));
+
+        jbJarEntryCertificates = new JButton(res.getString("DViewSignedJar.jbJarEntryCertificates.text"));
+
+        PlatformUtil.setMnemonic(jbJarEntryCertificates,
+                                 res.getString("DViewSignedJar.jbJarEntryCertificates.mnemonic").charAt(0));
+        jbJarEntryCertificates.setToolTipText(res.getString("DViewSignedJar.jbJarEntryCertificates.tooltip"));
+        jbJarEntryCertificates.setEnabled(false);
+        jbJarEntryCertificates.addActionListener(evt -> {
+            try {
+                CursorUtil.setCursorBusy(DViewSignedJar.this);
+                jarEntryCertificatesPressed();
+            } finally {
+                CursorUtil.setCursorFree(DViewSignedJar.this);
+            }
+        });
+
+        jbOK = new JButton(res.getString("DViewSignedJar.jbOK.text"));
+        jbOK.addActionListener(evt -> okPressed());
+
+        jbSignatures = new JButton(res.getString("DViewSignedJar.jbSignatures.text"));
+        jbSignatures.addActionListener(evt -> signaturesPressed());
+        PlatformUtil.setMnemonic(jbSignatures,
+                res.getString("DViewSignedJar.jbSignatures.mnemonic").charAt(0));
+        jbSignatures.setToolTipText(res.getString("DViewSignedJar.jbSignatures.tooltip"));
+
+        Container pane = getContentPane();
+        pane.setLayout(new MigLayout("insets dialog, fill", "[right]unrel[]", ""));
+
+        pane.add(jlVerifyStatus, "");
+        pane.add(jtfVerifyStatus, "sgx, wrap");
```

Comment:
It would be nice if the status text field extended over the entire width of the dialog:
<img width="614" height="344" alt="grafik" src="https://github.com/user-attachments/assets/3b29f060-7a1b-48ec-96a4-b5973a773bee" />

Replacing "sgx" with "growx" should do this.

Using a sizegroup ("sgx") only works when other rows are in the same sizegroup.